### ISSUE TITLE
Problems with tldr on IPv6-only devices

### ISSUE THREAD

**Issue by marek22k** [tldr-pages/tldr#10277]
### Command description

Hello,

in order for TLDR clients to obtain information, they would have to connect to GitHub. However, GitHub is IPv4-only. Devices or networks that are IPv6-only cannot access TLDR. Are there any plans to support IPv6-only nodes? For example by creating a mirror somewhere or moving the project to a free non-proprietary dual-stack git hosting platform?

### OS

- [ ] Android
- [ ] Linux
- [ ] macOS
- [ ] sunOS
- [ ] Windows

**Comment by pixelcmtd** [comment]
GitHub doesn't have IPv6? That sounds weird and should be reported to them, but at least for me they seem to have it:

```
$ q AAAA tldr-pages.github.io
tldr-pages.github.io. 56m45s AAAA 2606:50c0:8001::153
tldr-pages.github.io. 56m45s AAAA 2606:50c0:8003::153
tldr-pages.github.io. 56m45s AAAA 2606:50c0:8000::153
tldr-pages.github.io. 56m45s AAAA 2606:50c0:8002::153
```

I don't actually remember who's responsible for `tldr.sh` and don't know whether it's also backed by GH Pages. If necessary, we could host it somewhere else with IPv6 support as a workaround.

**Comment by kbdharun** [comment]
@pixelcmtd Regarding this we had a discussion in the chatroom today. I would recommend checking it out [here](https://matrix.to/#/!zXiOpjSkFTvtMpsenJ:gitter.im/$WKeCxaehnVF_GCo6G8XJiJYcfcAKbWYH-8jO5pqpDKg?via=gitter.im&via=matrix.org&via=one.ems.host).

Also, GitHub pages, release and raw user content support IPv6 but not the domain and API itself. This one is a highly requested issue but GitHub doesn't seem interested in adding it, the best method to manage this is to have a bidirectional mirror with GitLab (more info on chatroom).

**Comment by marek22k** [comment]
@kbdharun Perhaps the label "community" or "decision" would also be useful?

**Comment by Managor** [comment]
Has Github added this?
